modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -2197,6 +2197,62 @@ definition {
 			locator1 = "DocumentsAndMedia#DESCRIPTIVE_LIST_FOLDER_TITLE");
 	}
 
+	@description = "LPS-176828 - Verify that the Formula Builder has same structure as Expression Builder and contains only the principal math operations"
+	@priority = 5
+	test CanSeeAllOperations {
+		task ("Given: an Object") {
```

Comment:
Move the steps to add the formula field to this task. Change the wording to:
`Given a custom object with a formula field`